src/screens/Ticketing/FareContracts/Details/OrderDetails.tsx

```diff
@@ -2,13 +2,21 @@ import {FareContract} from '@atb/ticketing';
 import {FareContractTexts, useTranslation} from '@atb/translations';
 import {View} from 'react-native';
 import ThemeText from '@atb/components/text';
-import {formatToLongDateTime} from '@atb/utils/date';
+import {fullDateTime} from '@atb/utils/date';
 import {fromUnixTime} from 'date-fns';
 import _ from 'lodash';
 import React from 'react';
 import {StyleSheet} from '@atb/theme';
 
-const OrderDetails = ({fareContract}: {fareContract: FareContract}) => {
+const OrderDetails = ({
+  fareContract,
+  validFrom,
+  validTo,
+}: {
+  fareContract: FareContract;
+  validFrom: Date;
+  validTo: Date;
```

Comment:
A little illogical parameter list here as it mixes `FareContract` with information contained within it. I think it's better to retrieve `validFrom` and `validTo` from the `fareContract.travelRight[0]`. Don't need the `isPreActivatedTravelRight` check as this is already handled before.